features/banner.form.feature

```diff
@@ -4,24 +4,38 @@
 Feature: Checks wikimedia.de fundraising form functionality in banners for Wikipedia
 
   Background:
+    When I am on a random Wikipedia article page and provide a B15WMDE_bulb_prototype
+    And WMDE_Banner becomes visible
 
-  Scenario Outline: Checks if the form switches the interval options
-    When I am on a random Wikipedia article page and provide a <banner_code>
-    And The <banner_div_id> should become visible
-    And I wait a second
+  Scenario: Checks if the banner can be closed and the hidecookie is set
+    When WMDE_Banner becomes visible
+    And I reset the hide banner cookie centralnotice_wmde15_hide_cookie
+    And I click the banner close button
+    Then The WMDE_Banner banner should not be visible
+    And The hide banner cookie centralnotice_wmde15_hide_cookie should be set
+
+  Scenario: Checks if the form switches the interval options
     And I click the regularly interval option
-    Then Regularly details shows
+    Then The regularly details shows
```

Comment:
I think the case of `regularly` should be also changed in `features/senstive.ui.feature`. I had one test not run there due to undefined `Then Regularly details shows` step.